buildpipeline/Core-Setup-PortableLinux-x64.json

```diff
@@ -222,6 +394,33 @@
     },
     "NUGET_SYMBOLS_FEED_URL": {
       "value": "https://dotnet.myget.org/F/dotnet-core/symbols/api/v2/package"
+    },
+    "REPO_ID": {
+      "value": "PassedViaPipeBuild"
+    },
+    "REPO_USER": {
+      "value": "dotnet"
+    },
+    "REPO_PASS": {
+      "value": "PassedViaPipeBuild"
+    },
+    "REPO_SERVER": {
+      "value": "azure-apt-cat.cloudapp.net"
+    },
+    "DockerBuildEnvironmentVariables": {
```

Comment:
The name is a little confusing since it is not used for the `Run build in Docker container` step